pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/NamespacesBase.java

```diff
@@ -281,16 +288,19 @@ protected void internalDeleteNamespace(AsyncResponse asyncResponse, boolean auth
             if (log.isDebugEnabled()) {
                 log.debug("Found topics on namespace {}", namespaceName);
             }
-            boolean hasNonSystemTopic = false;
-            for (String topic : topics) {
+            for (String topic : partitionedTopicNames) {
```

Comment:
Can we use one loop?
It looks basically the same

Reply:
Already fixed